setup.py

```diff
@@ -72,15 +72,15 @@
         "Operating System :: POSIX",
         "Programming Language :: Python",
         "Programming Language :: Python :: 3",
-        "Programming Language :: Python :: 3.7",
         "Programming Language :: Python :: 3.8",
         "Programming Language :: Python :: 3.9",
         "Programming Language :: Python :: 3.10",
+        "Programming Language :: Python :: 3.11",
         "Programming Language :: Python :: Implementation :: CPython",
         "Programming Language :: Python :: Implementation :: PyPy",
     ],
-    # Requires Python 3.7+
-    python_requires=">=3.7",
+    # Requires Python 3.8+
+    python_requires=">=3.8",
```

Comment:
Do we want this change?
This may cause confusion for users, as `pip install fprime-tools` will install older versions without warning if running Python 3.7. But I don't think there is a better solution. The alternative is to keep it in, which may cause errors when running pip install.

Reply:
When we break the bootstrapping script into its own entity that can support multiple versions of python and users of (deprecated) 3.7 could then install older versions of F´  with supported packages.